### PR TITLE
Modify makesseeds.py to work with "protx list valid 1" instead of "masternode list

### PR DESCRIPTION
**Pull request by codablock**
This allows better filtering for MN owners with multiple MNs. This commit
also removes some unsupported fields, e.g. "protocol", "lastseen", ...

### CODE REVIEW

**Comment by nmarley** [comment]
Can you elaborate more on the purpose of filtering MNs from the same owner? What is this intended to achieve?

**Comment by codablock** [comment]
@nmarley It's the same purpose as with filtering by ASN. We don't want to have multiple entries pointing to multiple nodes of the same person. In Bitcoin ASN filtering was more critical as it protected against too many malicious nodes getting into the hardcoded seeds list. For us, it's more like a protection against large parts of networks or multi-MN owners going down.

File: contrib/seeds/README.md [hunk 1]
```diff
@@ -3,14 +3,16 @@
...
 (see [src/chainparamsseeds.h](/src/chainparamsseeds.h) and other utilities in [contrib/seeds](/contrib/seeds)).
 
-Be sure to update `MIN_PROTOCOL_VERSION` in `makeseeds.py` to include the current version.
+The seeds compiled into the release are created from the current protx list, like this:
 
-The seeds compiled into the release are created from the current masternode list, like this:
-
-    dash-cli masternodelist full > mnlist.json
-    python3 makeseeds.py < mnlist.json > nodes_main.txt
+    dash-cli protx list 1 1185193 > protx_list.json
```
Comment (UdjinM6) [inline contrib/seeds/README.md]:
```suggestion
    dash-cli protx list valid 1 1185193 > protx_list.json
```

**Comment by nmarley** [comment]
Ok, that makes sense (kinda). ASN is more physical, e.g. if they have the same ASN they are in the same physical space, so guards against data center going down. Filtering by owner is not necessarily that at all, they can be geographically distributed. Filtering for malicious nodes makes sense, but malicious MNOs who are aware of this can split out collateral / payouts and avoid this precaution.

File: contrib/seeds/README.md (hunk 1 above)
Comment (codablock) [reply contrib/seeds/README.md]: squashed into original commit

**Pull request merged**